app/views/setup/discovery.html.slim

```diff
@@ -0,0 +1,34 @@
+.alert.alert-info.alert-dismissible role="alert"
+  button.close[type="button" data-dismiss="alert" aria-label="Close"]
+    span[aria-hidden="true"]
+      | &times;
+  i.fa.fa-4x.pull-left aria-hidden="true"
+  span
+    | After choosing the master and clicking "Bootstrap cluster" all other nodes will be set to the minion role
+h1 Nodes
```

Comment:
I wouldn't worry too much about the wording here, as we can't add nodes from the UI anyways. it needs to be done in the background by the user, and then the nodes are just popping up